src/NodaTime.Test/TimeZones/BclDateTimeZoneTest.cs

```diff
@@ -67,14 +89,14 @@ public void DisplayName(TimeZoneInfo windowsZone)
         /// This test catches situations where the Noda Time representation doesn't have all the
         /// transitions it should; AllZoneTransitions may pass not spot times when we *should* have
         /// a transition, because it only uses the transitions it knows about. Instead, here we
-        /// check each day between 1st January 1950 and 1st January 2050. We use midnight UTC, but
+        /// check each week between 1st January 1950 and 1st January 2050. We use midnight UTC, but
```

Comment:
It's actually 2050 or 2037, isn't it? (Not sure how precise you want to be here.)

Reply:
Updated.